_posts/en/newsletters/2019-09-25-newsletter.md

```diff
@@ -78,7 +78,24 @@ answers made since our last update.*
 endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME
+- [Why does this coinbase transaction have two op_return outputs?]({{bse}}90127)
+Murch outlines that while there are some restrictions on coinbase transactions,
+there are no special script or quantity restrictions regarding coinbase
+transaction outputs.
```

Comment:
Murch's full answer seems a bit confused to me by noting a place where generation transaction outputs are restricted (the case where the block contains segwit transactions) and then saying that there are no restrictions.  Your summary only quotes the later part and so (it seems to me) twists the answer from confusing to wrong.  I left a comment for Mark on the SE; maybe this could say something like "....regarding coinbase transaction outputs, save that any blocks containing segwit spends must have an output matching the [BIP141 commitment structure](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#Commitment_structure)."

Reply:
Pushed a change based on Murch's revised answer and your feedback. Feel free to tweak!